java/.ci/Jenkinsfile

```diff
@@ -769,7 +769,28 @@ pipeline {
                     }
                 }
             }
-            
+            stage('Parallel Execution 10'){
+                                        parallel{
+                                            stage('PubSubLite To BigTable') {
+                                                steps{
+                                                    sh '''
+                                                        export GCP_PROJECT=yadavaja-sandbox
+                                                        export REGION=us-west1
+                                                        export GCS_STAGING_LOCATION="gs://python-dataproc-templates-temp"
```

Comment:
Please remove these, as they are already present in the Jenkins Global Variables.

Reply:
Removed and committed